src/Frame.cc

```diff
@@ -217,3 +218,18 @@ sdf::ElementPtr Frame::Element() const
 {
   return this->dataPtr->sdf;
 }
+
+/////////////////////////////////////////////////
+sdf::ElementPtr Frame::ToElement() const
+{
+  sdf::ElementPtr elem(new sdf::Element);
+  sdf::initFile("frame.sdf", elem);
+
+  elem->GetAttribute("name")->Set(this->dataPtr->name);
+
+  if (!this->dataPtr->attachedTo.empty())
+    elem->GetAttribute("attached_to")->Set(this->dataPtr->attachedTo);
+
+  elem->GetElement("pose")->Set(this->dataPtr->pose);
```

Comment:
I recommend copying the code from `Link::ToElement` that handles the relative_to attribute:

https://github.com/gazebosim/sdformat/blob/main/src/Link.cc#L655-L662

Reply:
Done in 76d3794